__tests__/app.test.js

```diff
@@ -142,4 +142,50 @@ describe('/api/articles/:article_id/comments', ()=>{
       expect(response.body.msg).toBe('Not Found');
     });
   })
-})
\ No newline at end of file
+})
+
+describe('Post request /api/articles/:article_id/comments',() => {
+  test('POST:201 adds a comment to specified article with correct shape',() => {
+    const newComment = {
+      username:'butter_bridge',
+      body: 'Immortal'
+    }
+    return request(app)
+    .post('/api/articles/3/comments')
+    .send(newComment)
+    .expect(201)
+    .then((response)=> {
+      expect(response.body.comment.author).toBe(newComment.username)
+      expect(response.body.comment.body).toBe(newComment.body)
+      expect(response.body.comment)
+    })
+  })
+  test('POST:400 responds with an error message when given an invalid id', () => {
+
+    const newComment = {
+      username:'butter_bridge',
+      body: 'Immortal'
+    }
+    return request(app)
+    .post('/api/articles/iNeedPizza/comments')
+    .send(newComment)
+    .expect(400)
+    .then((response)=> {
+      expect(response.body.msg).toBe('Bad request')
+  })
+  })
+  test('POST:404 responds with an error message when given a valid but non-existent id', () => {
+
+    const newComment = {
+      username:'butter_bridge',
+      body: 'Immortal'
+    }
+    return request(app)
+    .post('/api/articles/07734/comments')
+    .send(newComment)
+    .expect(404)
+    .then((response)=> {
+      expect(response.body.msg).toBe('Not Found');
+  })
```

Comment:
There are a couple more tests you should have for posting a comment
- Status 400, missing required field(s), e.g. no username or body properties
- Status 404, username does not exist